u-mode-api/src/cert.rs

```diff
@@ -0,0 +1,28 @@
+use attestation::MSMT_REGISTERS;
```

Comment:
this needs a copyright / license thingy at the top of the file

Reply:
Done.